### DISCUSSION TITLE
Check for future Prices to select cities

### DISCUSSION THREAD

**Discussion by netwavetech** [jdholtz/auto-southwest-check-in#218] [Ideas]
Add the option to add certain routes to check for low price sales.  I.e. I fly from Nashville (BNA) to Ft Lauderdale (FLL) a few times a year when the price drops low enough.  Maybe set a low price point.  If it drops below $89, etc?  But even if it see a drop from the last week.

**Comment by jdholtz** [comment]
Hm, my initial thoughts are that this seems to far out of the scope of this project. This project simply takes in a list of scheduled flights from your account (or reservation information the user provides) and monitors those. Adding a separate watcher for flights in general would be a very different concept than this script and almost none of the code would be shared between with this feature. I think this would be a good idea for a separate project, which I could make if people want this type of thing (although not right now as I don't have the time for it, but maybe this spring or summer).
